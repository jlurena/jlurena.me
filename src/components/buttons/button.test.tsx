import { render } from '@testing-library/react';
import { describe, it, expect } from 'vitest';
import  Button from './button';

describe('<Button />', () => {
  it('renders correctly (snapshot)', () => {
    const { asFragment } = render(
      <Button className="someclass">
        <p>Hello</p>
      </Button>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
