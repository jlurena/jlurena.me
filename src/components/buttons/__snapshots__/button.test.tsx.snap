// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Button /> > renders correctly (snapshot) 1`] = `
<DocumentFragment>
  <button
    class="someclass _btn_9c2c4d"
    type="button"
  >
    <p>
      Hello
    </p>
  </button>
</DocumentFragment>
`;
